config/regulations-site/handler.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import os
+import sys
+
+def reg_site(event, context):
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "regulations.settings.dev") 
+    try:
+        from django.core.management import excecute_from_command_line
+    except ImportError:
+        try:
+            import django
+        except ImportError:
+            raise ImportError(
+                "Couldn't import Django.
+            )
+execute_from_command_line(sys.argv)
```

Comment:
Make this match the other handler (or vice versa)
```suggestion
execute_from_command_line(['manage.py', 'runserver'])
```

Reply:
I wrote it (sys.argv) for that handler.py because manage.py for regulations-site has (sys.argv) instead of ('manage.py', 'runserver'), but I'll change it.